### DISCUSSION TITLE
Set up NixOS Distrobox + GUI

### DISCUSSION THREAD

**Discussion by aidanhadley** [89luca89/distrobox#1181] [Q&A]
this this is awesome, and very useful.
i want to try about NixOS, but i don't want to use a virtual machine due to resources on my computer, and i don't have another i can try it on. i'm thinking that distrobox is the best way to do it. i'm wanting to do 2 things:
1. install nixos on distrobox
2. have a gui that i can access through gdm
the second one is more niche, but i know it's somewhat possible because of [this article](https://cloudyday.tech.blog/2022/05/14/distrobox-is-awesome/), and i want to know how to do it on my machine.
i'm currently running endeavouros (essentially arch linux). and i'll be happy to provide more info

**Comment by SerJaimeLannister** [comment]
Yes I am interested in the same as well!
